zap/src/main/java/org/zaproxy/zap/extension/api/ApiGeneratorUtils.java

```diff
@@ -76,10 +76,6 @@ public static List<ApiImplementor> getAllImplementors() {
         api.addApiOptions(new OptionsParamCheckForUpdates());
         imps.add(api);
 
-        api = new SpiderAPI(null);
-        api.addApiOptions(new SpiderParam());
-        imps.add(api);
-
```

Comment:
This was necessary to make the build happy, but likely isn't the right option?

Reply:
It's fine, the spider API should be generated from the add-on once things are merged.